app/models/channel.rb

```diff
@@ -98,15 +109,20 @@ def promo_enabled?
   end
 
   def verification_started!
-    update!(verified: false, verification_status: 'started', verification_details: nil)
+    update!(verified: false, verification_status: STARTED, verification_details: nil)
   end
 
   def verification_failed!(details = nil)
     # Clear changes so we don't bypass validations when saving without checking them
     self.reload
 
+    if manual_verification_running
+      PublisherMailer.verification_failed(self).deliver_later
+      self.update(manual_verification_running: false)
```

Comment:
minor: `#save!` runs below so `#update` could be replaced with a simple assignment.